.travis.yml

```diff
@@ -0,0 +1,46 @@
+language: php
+sudo: false
+dist: trusty
+
+php: 7.1
+
+os: linux
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+stages:
+  - check coding style
+  - static code analysis
+
+before_install:
+  - phpenv config-rm xdebug.ini || echo "xdebug not available"
+  - echo 'opcache.enable=1' >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - echo 'opcache.enable_cli=1' >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - travis_retry composer self-update
+
+install: travis_retry composer install --optimize-autoloader --prefer-dist --no-progress --no-interaction --no-suggest -vv
+
+script: skip
+
+jobs:
+  fast_finish: true
+  include:
+    - stage: check coding style
```

Comment:
I don't think it's necessary to use both php-cs-fixer and phpcs.  Please remove phpcs.

Reply:
fixed